functions/serverless-functions-getting-started.adoc

```diff
@@ -13,7 +13,10 @@ Function lifecycle management includes creating, building, and deploying a funct
 
 Before you can complete the following procedures, you must ensure that you have completed all of the prerequisite tasks in xref:../functions/serverless-functions-setup.adoc#serverless-functions-setup[Setting up {FunctionsProductName}].
 
+// Creating functions
 include::modules/serverless-create-func-kn.adoc[leveloffset=+1]
+include::modules/odc-creating-functions.adoc[leveloffset=+1]
```

Comment:
If your new procedure is an alternative to the existing, previous procedure (using `kn`), then I'd suggest considering insertion of a brief assembly that would include both as subsections. I mean something like this:

```
:_content-type: ASSEMBLY
= Creating functions

A concise intro with two hyperlinked bullet points ...:
* xref:modules/serverless-create-func-kn.adoc[... the Knative (kn) CLI]
* xref:modules/odc-creating-functions.adoc[... the Developer perspective of the Red Hat OpenShift Serverless web console]

include::modules/serverless-create-func-kn.adoc[leveloffset=+1]
include::modules/odc-creating-functions.adoc[leveloffset=+1]
```

For example:

`= Creating a function by using the CLI` in `modules/serverless-create-func-kn.adoc`

`= Creating a function by using the web console` in `modules/odc-creating-functions.adoc`

And then these file names could be tweaked and aligned with this distinction

Reply:
I thought about doing this but didn't want to increase the scope of the issue too much at the time. Might revisit it though.

Reply:
I won't create a new assembly for this, since I would need to either use nested assemblies or dramatically change the doc structure in the topic map (this would be an ask for the serverless team), but I did change the include levels of the modules and add a heading to group them.